src/everest/simulator/everest_to_ert.py

```diff
@@ -25,27 +24,19 @@ def extract_summary_keys(ever_config: EverestConfig) -> list[str]:
 
     data_keys = everest.simulator.DEFAULT_DATA_SUMMARY_KEYS
     field_keys = everest.simulator.DEFAULT_FIELD_SUMMARY_KEYS
-    well_sum_keys = everest.simulator.DEFAULT_WELL_SUMMARY_KEYS
+    well_keys = [
+        f"{sum_key}:*" for sum_key in everest.simulator.DEFAULT_WELL_SUMMARY_KEYS
+    ]
```

Comment:
Can you check usage of
`_EXCLUDED_TARGET_KEYS`, `_DEFAULT_WELL_TARGET_SUMMARY_KEYS`, `DEFAULT_WELL_SUMMARY_KEYS`, `_DEFAULT_WELL_SUMMARY_KEYS` ?
Perhaps some of this can be simplified even more?

This processing done here;
```
well_keys = [
        f"{sum_key}:*" for sum_key in everest.simulator.DEFAULT_WELL_SUMMARY_KEYS
    ]
```
seems like it is also done in `test_res_initialization.py` ?